doc/protocols/sigprefixlist/MappingService.md

```diff
@@ -0,0 +1,47 @@
+# Mapping Service
+The Mapping Service (MS) acts as a bridge between the Publishing Infrastructure and the SIGs. It is also a trusted entity for the ISD it is deployed in and should be deployed in Core ASs only. 
+
+It is responsible for forming lists of mappings it receives from downstream ASs and submitting them to the Publishing Infrastructure. It pulls mapping lists for other ISDs published in the Publishing Infrastructure and stores it and responds to SIGs that require the mappings. 
+
+## Deployment 
+To deploy a Mapping Service instance run the following command from [go/ms](../../../go/ms) folder
+```
+go run main.go -config <path_to_config_file>
+```
+For more information on the configuration see 
+[Configuration](#Configuration)
+
+The Mapping Service connects to a [PLN](./PublishingListNode.md) instance. The PLN instance must be running when starting a MS instance.
+
+## Configuration
+The sample configuration file can be generated using 
+```
+go run main.go -help-config
+```
+from [go/ms](../../../go/ms)
+
+## General Structure
+It reuses existing packages to build up the service
+- [go/lib/env](../../../go/lib/env): Is used for configuration and setup of the service.
```

Comment:
It would be nice if capitalization, punctuation, etc. of different lists were consistent.

Reply:
I changed them to be consistent